### PR TITLE
Fix truncated text about Cosmos DB regions

### PR DESCRIPTION
**Pull request by jawn**

### CODE REVIEW

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/checklist/resiliency.md](https://github.com/jawn/architecture-center/blob/patch-2/docs/checklist/resiliency.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/checklist/resiliency?branch=pr-en-us-256) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C10%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201710100926439347-256%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=6k5Z%2BziEOrgLV%2FUXzjg%2B9LeA%2B9t%2BzGHo9AENlb6DOTI%3D&st=2017-10-10T09%3A22%3A31Z&se=2017-11-10T09%3A27%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/checklist/resiliency.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Resiliency checklist
 description: Checklist that provides guidance for resiliency concerns during design.
 author: petertaylor9999
-ms.date: 03/24/2017
+ms.date: 03/24/201
```
Comment (MikeWasson) [inline docs/checklist/resiliency.md]: @jawn Minor thing, but can you revert this change? Then I'll merge. The other change looks good. Thanks!

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/checklist/resiliency.md](https://github.com/jawn/architecture-center/blob/patch-2/docs/checklist/resiliency.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/checklist/resiliency?branch=pr-en-us-256) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C10%5C10%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201710101459181033-256%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=uUIRBRceHaEOuslXh5Wf1KJoCQuXyX32Ers9Bq1Dl9Y%3D&st=2017-10-10T14%3A55%3A28Z&se=2017-11-10T15%3A00%3A28Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/checklist/resiliency.md (hunk 1 above)
Comment (jawn) [reply docs/checklist/resiliency.md]: Done, updated the PR.

**Comment by MikeWasson** [comment]
Looks good - thanks for the PR!

**Pull request merged**